CHANGELOG.md

```diff
@@ -1,32 +1,29 @@
+### Fixed
+- Log messages in RP shown without ANSI characters
```

Comment:
Could you please mention `RP_DESCRIPTION` & `RP.MODE` env variables support in `Added` section here?